Makefile

```diff
@@ -60,18 +60,18 @@ EFM8_LOAD_BAUD	?= 115200
 define MAKE_OBJ
 OBJS += $(1)_$(2)_$(3)_$(4)_$(VERSION).OBJ
 $(OUTPUT_DIR)/$(1)_$(2)_$(3)_$(4)_$(VERSION).OBJ : $(ASM_SRC) $(ASM_INC)
-	$(eval _ESC			:= $(1))
-	$(eval _ESC_INT		:= $(shell printf "%d" "'${_ESC}"))
-	$(eval _ESCNO		:= $(shell echo $$(( $(_ESC_INT) - 65 + 1))))
-	$(eval _MCU_48MHZ	:= $(subst L,0,$(subst H,1,$(subst X,2,$(2)))))
-	$(eval _DEADTIME	:= $(3))
-	$(eval _PWM_FREQ	:= $(subst 24,0,$(subst 48,1,$(subst 96,2,$(4)))))
-	$$(eval _LST		:= $$(patsubst %.OBJ,%.LST,$$@))
+	$(eval _ESC				:= $(1))
```

Comment:
Could you remove the added whitespaces please?

Reply:
Done